modules/serverless-logic-asyncapi-consuming-event.adoc

```diff
@@ -1,15 +1,15 @@
 // Module included in the following assemblies:
-//
+// * serverless-logic/serverless-logic-managing-events/serverless-logic-configuring-asyncapi
 
 :_mod-docs-content-type: PROCEDURE
-[id="serverless-logic-asyncAPI-consuming-event_{context}"]
+[id="serverless-logic-asyncapi-consuming-event_{context}"]
 = Consuming an event
 
 To enable seamless event consumption, define the Kafka server, subscription channel, and a function that consumes the event, and then invoke the function in your workflow. When your workflow reaches the function, it pauses and waits for the specific event from the Kafka topic before continuing.
 
 .Prerequisites
-* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {ocp-product-title}.
-* You have created your {ServerlessLogicProductName} project.
+
+* You have access to an {ServerlessLogicProductName} project with the appropriate roles and permissions to create applications and other workloads in {ocp-product-title}.
 * You have created the `asyncAPI.yaml` file and your project `classpath`.
 * Your project includes necessary dependencies for AsyncAPI and Kafka.
```

Comment:
How do I add these into my project?

Reply:
Seems like the AsyncAPI dependencies are in by default, but kafka is not `io.quarkus:quarkus-smallrye-reactive-messaging-kafka` and further more this guide should not be using kafka imho